{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/metrics.py

```diff
@@ -0,0 +1,33 @@
+{% if cookiecutter.monitoring == "y" %}
+import glob
+import os
+
+import prometheus_client
+from django_prometheus.exports import ExportToDjangoView
+from django.http import HttpResponse
+from prometheus_client import multiprocess
+
+
+class RecursiveMultiProcessCollector(multiprocess.MultiProcessCollector):
+    """A multiprocess collector that scans the directory recursively"""
+    def collect(self):
+        files = glob.glob(os.path.join(self._path, '*.db'), recursive=True)
+        return self.merge(files, accumulate=True)
+
+
+env_var_name = 'PROMETHEUS_MULTIPROC_DIR'
+
+
+def metrics_view(request):
+    """Exports metrics as a Django view"""
+    if os.environ.get(env_var_name) or os.environ.get(env_var_name.lower()):
```

Comment:
I left support for both upper and lowercase envvar, because both prometheus-client and Django-prometheus support them. Although lower case is deprecated, so I guess we can scrap it.

Reply:
now it is the way it should be: it's either uppercase or error